web/website/content/faq.md

```diff
@@ -6,11 +6,16 @@ Here are some of the most common questions we hear. Have something else you'd
 like to ask? Pop by our [Discord](https://discord.com/invite/eQcfaCmsNc) and ask
 away!
 
-{{< faq "Cool story Hansel, but what can I actually do with PRQL now?" >}}
+{{< faq "Cool story Hansel, but what can I actually do with PRQL?" >}}
 
-We're still early, and the opportunities for using PRQL are focused on two
-integrations:
+PRQL is a great tool for building SQL queries. In addition, we are focusing on
```

Comment:
I think we should try and avoid any normative descriptions such as "great" on our own site

Reply:
Good point. Rewording.